backend/integrations/discord/bot.py

```
@@ -66,13 +79,47 @@ async def on_message(self, message):
+        if thread_id:
+            thread = self.get_channel(int(thread_id))
+            if thread:
+                await thread.send("I'm processing your request, please hold on...")
+
+        # 🔀 execution split
+        if self.queue_manager:
+            await self._handle_devrel_with_queue(message, triage_result, thread_id)
+        else:
+            await self._handle_devrel_direct(message, triage_result, thread_id)
+
+    async def _handle_devrel_direct(self,message, triage_result, thread_id):
```

Ruff (0.14.14)

[warning] Unused method argument: `triage_result`

(ARG002)